docs/controllers/extendedjob.md

```diff
@@ -20,63 +24,96 @@ There are two different kinds of `ExtendedJob`:
 - **one-offs**: automatically runs once after it's created
 - **errands**: needs to be run manually by a user
 
-![flow-extended-job](quarks_deployment_flow-ExtendedJob.png)
+## ExtendedJob Component
 
-## Features
+The **ExtendedJob** component is a categorization of a set of controllers, under the same group. Inside the **ExtendedJob** component, we have a set of 2 controllers together with 2 separate reconciliation loops.
 
-### Errand Jobs
+The following, is a **ExtendedJob** component diagram that covers the set of controllers it uses.
```

Comment:
Similar, all over the place. Using a `Figure X is` feels better in all these places. No further separate comment anymore.